source/docs/configuration.md

```diff
@@ -70,6 +70,7 @@ Setting | Description | Default
 `highlight.auto_detect` | Enable auto-detection if no language is specified | `false`
 `highlight.line_number` | Display line number | `true`
 `highlight.tab_replace` | Replace tabs by n space(s); if the value is empty, tabs won't be replaced | `''`
+`use_date_for_updated` | Use the date of the post as date of update if no updated date is provided. Useful when the file's dates are meaningless (especially when using git) | `true`  
```

Comment:
Shouldn't this be under `Date / Time format` section? It's under similar section in [default_config.js](https://github.com/tomap/hexo/blob/677f5b86069a3395e630d749bd3dd7c3a30d3ba8/lib/hexo/default_config.js#L49).